### PR TITLE
Preparation for 1.4.1 release

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: src/NodaTime.Test/Text/ZonedDateTimePatternTest.cs [hunk 1]
```diff
@@ -197,6 +197,9 @@ public class ZonedDateTimePatternTest : PatternTestBase<ZonedDateTime>
             // Standard patterns with a time zone provider
             new Data(2013, 01, 13, 15, 44, 30, 0, TestZone1) { Pattern = "G", Text = "2013-01-13T15:44:30 ab (+02)", Culture = Cultures.FrFr },
             new Data(2013, 01, 13, 15, 44, 30, 90, TestZone1) { Pattern = "F", Text = "2013-01-13T15:44:30.09 ab (+02)", Culture = Cultures.FrFr },
+
+            // Issue981
+            new Data(1906, 8, 29, 20, 58, 32, 0, DateTimeZoneProviders.Tzdb["Etc/GMT-12"]) { Text = "1906-08-29T20:58:32 Etc/GMT-12 (+12)", Pattern = "uuuu'-'MM'-'dd'T'HH':'mm':'ss;FFFFFFFFF z '('o<g>')'", ZoneProvider = DateTimeZoneProviders.Tzdb },
```
Comment (malcolmr) [inline src/NodaTime.Test/Text/ZonedDateTimePatternTest.cs]: We don't support `FFFFFFFFF` in 1.4.x; this'll need to be `FFFFFFF`.

(This is why Travis is failing.)

Comment (jskeet) [reply src/NodaTime.Test/Text/ZonedDateTimePatternTest.cs]: Doh - thanks, well spotted. (Hadn't even checked Travis.)

Comment (jskeet) [reply src/NodaTime.Test/Text/ZonedDateTimePatternTest.cs]: (Although I did think I'd run the tests. There were fewer than I'd expected, but I thought that was just that we had fewer tests in 1.x. Maybe not. Will investigate.)

**Comment by jskeet** [comment]
It needs to be yyyy instead of uuuu as well, for 1.4.1 :) (Have managed to get all tests running under VS.)

**Comment by jskeet** [comment]
Thanks. Will see if I can do the releases tomorrow afternoon.

**Pull request merged**